sample/source/conf.py

```diff
@@ -1,6 +1,9 @@
 """
 Sample ``conf.py``.
 """
+import os
```

Comment:
* What does this change achieve?
  * Is that relevant to this PR?

If it is required and relevant:

* Can a comment be added so that maintainers understand why this is here?
* Can this be done with [`Pathlib`](https://docs.python.org/3/library/pathlib.html)?

Reply:
I've deleted them - it's not relevant to this PR.

I was trying to build the sample docs with the command ``sphinx-build``, but got an error without those lines. So now I've tried building the docs with the command ``make build-sample`` and it seems to work as expected without those lines.